.github/workflows/llm_performance_tests_stable_version.yml

```diff
@@ -69,87 +69,92 @@ jobs:
           bash python/llm/test/run-llm-install-tests.sh
 
-      - name: Test on xpu
+      - name: Test on xpu (int4)
         shell: bash
         run: |
           source /home/arda/intel/oneapi/setvars.sh
           export USE_XETLA=OFF
           export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
-          mv python/llm/test/benchmark/stable-version-arc-perf-test.yaml python/llm/dev/benchmark/all-in-one/config.yaml
+          mv python/llm/test/benchmark/stable-version-arc-perf-test-sym_int4.yaml python/llm/dev/benchmark/all-in-one/config.yaml
           cd python/llm/dev/benchmark/all-in-one
           # hide time info
           sed -i 's/str(end - st)/"xxxxxx"/g' run.py
-          # change csv name
-          sed -i 's/{today}/{today}_test1/g' run.py
           python run.py
-          # upgrade transformers for model Mistral-7B-v0.1
-          python -m pip install transformers==4.34.0
-          mv ../../../test/benchmark/arc-perf-transformers-434.yaml ./config.yaml
-          # change csv name
-          sed -i 's/test1/test2/g' run.py
-          python run.py
-          python ../../../test/benchmark/concat_csv.py
-          cp ./*.csv $CSV_SAVE_PATH
+          cp ./*.csv $CSV_SAVE_PATH/int4
+          rm ./*.csv
           cd ../../../test/benchmark
           python -m pip install pandas==1.5.3
-          python csv_to_html.py -f $CSV_SAVE_PATH
-          cd ../../dev/benchmark/all-in-one/
-          python ../../../test/benchmark/check_results.py -n 45
-          if [ ${{ github.event.schedule}} ]; then
-            curl -T ./*.csv ${LLM_FTP_URL}/llm/nightly_perf/gpu/
-          fi
-          
-  llm-performance-test-on-spr:
-    needs: llm-cpp-build
-    strategy:
-      fail-fast: false
-      matrix:
-        python-version: ["3.9"]
-    runs-on: [self-hosted, llm, spr01-perf]
-    env:
-      OMP_NUM_THREADS: 16
-      THREAD_NUM: 16
-      ANALYTICS_ZOO_ROOT: ${{ github.workspace }}
-    steps:
-      - uses: actions/checkout@v3
-
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v4
-        with:
-          python-version: ${{ matrix.python-version }}
-
-      - name: Install dependencies
-        shell: bash
-        run: |
-          python -m pip install --upgrade pip
-          python -m pip install --upgrade wheel
-          python -m pip install --upgrade omegaconf
-          python -m pip install --upgrade pandas
-          python -m pip install --upgrade einops
-          python -m pip install --upgrade tiktoken
-          python -m pip install --upgrade transformers_stream_generator
-
-      - name: Download llm binary
-        uses: ./.github/actions/llm/download-llm-binary
-
-      - name: Run LLM install (all) test
-        uses: ./.github/actions/llm/setup-llm-env
+          python csv_to_html.py -f $CSV_SAVE_PATH/int4 -b $CSV_SAVE_PATH/int4/transformer_int4_gpu-results-1baseline.csv -t 5.0
 
-      - name: Test on cpu
+      - name: Test on xpu (fp8)
```

Comment:
Do we need to put each precision as a separate job? Why not running the bash for multiple times in a single job?

Reply:
Can leave this for now, seems quite clear already.